KitePrintSDK/src/main/java/ly/kite/analytics/Analytics.java

```diff
@@ -0,0 +1,660 @@
+   *****************************************************/
+  public void trackSDKLoaded( String entryPoint )
+    {
+    JSONObject propertiesJSONObject = getPropertiesJSONObject();
+
+    try
+      {
+      propertiesJSONObject.put( JSON_PROPERTY_NAME_ENTRY_POINT, entryPoint );
+      }
+    catch ( JSONException je )
+      {
+      Log.e( LOG_TAG, "Error setting JSON properties", je );
+      }
+
+    trackEvent( EVENT_NAME_SDK_LOADED, propertiesJSONObject );
+    }
+
+
+  /*****************************************************
+   *
+   * Called when the user enters the product list screen.
+   *
+   *****************************************************/
+  public void trackProductSelectionScreenViewed()
+    {
+    trackEvent( EVENT_NAME_PRODUCT_SELECTION_SCREEN_VIEWED );
+    }
+
+
+  /*****************************************************
+   *
+   * Called when the user enters the product overview screen.
+   *
+   *****************************************************/
+  public void trackProductOverviewScreenViewed( Product product )
+    {
+    JSONObject propertiesJSONObject = getPropertiesJSONObject();
+
+    try
+      {
+      propertiesJSONObject.put( JSON_PROPERTY_NAME_PRODUCT_NAME, product.getName() );
+      }
+    catch ( JSONException je )
+      {
+      Log.e( LOG_TAG, "Error setting JSON properties", je );
+      }
+
+    trackEvent( EVENT_NAME_PRODUCT_OVERVIEW_SCREEN_VIEWED, propertiesJSONObject );
+    }
+
+
+  /*****************************************************
+   *
+   * Called when the user enters the product overview screen.
+   *
+   *****************************************************/
+  public void trackCreateProductScreenViewed( Product product )
+    {
+    JSONObject propertiesJSONObject = getPropertiesJSONObject();
+
+    try
+      {
+      propertiesJSONObject.put( JSON_PROPERTY_NAME_PRODUCT_NAME, product.getName() );
+      }
+    catch ( JSONException je )
+      {
+      Log.e( LOG_TAG, "Error setting JSON properties", je );
+      }
+
+    trackEvent( EVENT_NAME_CREATE_PRODUCT_SCREEN_VIEWED, propertiesJSONObject );
+    }
+
+
+  /*****************************************************
+   *
+   * Called when the user enters the shipping screen.
+   *
+   *****************************************************/
+  public void trackShippingScreenViewed( PrintOrder printOrder, String variant, boolean showPhoneEntryField )
+    {
+    JSONObject propertiesJSONObject = getPropertiesJSONObject();
+
+    addToJSON( printOrder, propertiesJSONObject );
+
+    try
+      {
+      propertiesJSONObject.put( JSON_PROPERTY_NAME_SHIPPING_SCREEN_VARIANT, variant );
+      propertiesJSONObject.put( JSON_PROPERTY_NAME_SHOW_PHONE_ENTRY_FIELD,  ( showPhoneEntryField ? JSON_PROPERTY_VALUE_YES : JSON_PROPERTY_VALUE_NO ) );
+      }
+    catch ( JSONException je )
+      {
+      Log.e( LOG_TAG, "Error setting JSON properties", je );
+      }
+
+    trackEvent( EVENT_NAME_CREATE_PRODUCT_SCREEN_VIEWED, propertiesJSONObject );
```

Comment:
I think you mean something like `trackEvent( EVENT_NAME_SHIPPING_SCREEN_VIEWED, propertiesJSONObject );`

where `EVENT_NAME_SHIPPING_SCREEN_VIEWED = "Shipping Screen Viewed"`